### PR TITLE
Change cloud sdk version to 276.0.0

### PR DESCRIPTION
**Pull request by wchamber**
I need to bump the cloud sdk version to get newer releases of the datastore emulator for CA-648.

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've followed [the instructions](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [x] I've updated the [FISMA documentation](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary. Documentation PRs only need 1 thumb.
- [x] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by gpolumbo-broad — APPROVED** [review]

**Comment by wchamber** [comment]
jenkins retest

**Comment by wchamber** [comment]
jenkins retest

File: docker/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ RUN apt-get update \
 # Install the Google Cloud SDK
 ENV HOME /
 ENV CLOUDSDK_PYTHON_SITEPACKAGES 1
-RUN wget -O google-cloud-sdk.tar.gz https://storage.googleapis.com/cloud-sdk-release/google-cloud-sdk-218.0.0-linux-x86.tar.gz \
+RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > google-cloud-sdk.tar.gz \
```
Comment (andy7i) [inline docker/Dockerfile]: is curl > wget? just wondering about the change

**Review by andy7i — APPROVED** [review]
seems reasonable 👍

File: docker/Dockerfile (hunk 1 above)
Comment (wchamber) [reply docker/Dockerfile]: No particular reason. Some example from google using curl.

**Review by dvoet — APPROVED** [review]

**Comment by wchamber** [comment]
I've changed this to fix it at 276.0.0 instead of latest. There was an google cloud sdk release today that seems to have lost pip. I think it's ok to fix to a version instead of staying at latest.

Still need to figure out the dev_appserver errors from imports - I expect Jenkins to still fail.

**Comment by wchamber** [comment]
Merging this into python3-feature branch instead. To bump the cloud sdk version, we need to stop using endpoints to remove some versioning issue with dependencies. We need flask to remove endpoints. We're keeping flask in the python3 feature branch.

**Pull request merged**